docs/components/dividers.md

```diff
@@ -0,0 +1,225 @@
+<!--docs:
+title: "Material dividers"
+layout: detail
+section: components
+excerpt: "A divider is a thin line that groups content in lists and layouts."
+iconId:
+path: /catalog/material-dividers/
+-->
+
+# Dividers
+
+A [divider](https://material.io/components/dividers) is a thin line that groups content in lists and layouts.
+
+![Example divider: full-bleed dividers](assets/dividers/Dividers_hero.png)
+
+## Contents
+
+- [Contents](#contents)
+- [Making dividers accessible](#making-dividers-accessible)
+- [Anatomy and key properties](#anatomy-and-key-properties)
+- [Types](#types)
+- [Theming dividers](#theming-dividers)
+
+## Making dividers accessible
+
+Flutter's APIs support accessibility setting for large fonts, screen readers, and sufficient contrast. For more information, go to Flutter's [accessibility](https://flutter.dev/docs/development/accessibility-and-localization/accessibility) and [internationalization](https://flutter.dev/docs/development/accessibility-and-localization/internationalization) pages.
+
+For more guidance on writing labels, go to [our page on how to write a good accessibility label](https://material.io/design/usability/accessibility.html#writing).
+
+## Anatomy and key properties
+
+The following shows the anatomy of a divider:
+
+![Divider anatomy](assets/dividers/dividers_anatomy.png)
+
+### Divider
+
+| &nbsp; | Property |
+|---|---|
+| Color | Within the `Divider` widget use the `color` parameter |
+| Space | Within the `Divider` widget use the `height` parameter. Within the `VerticalDivider` widget use the  `width` parameter.|
+| Indent | Within the `Divider` widget use the `indent` parameter |
+| End Indent | Within the `Divider` widget use the `endIndent` parameter |
+| Thickness | Within the `Divider` widget use the `thickness` parameter |
+
+## Types
+
+There are 3 types of dividers:
+
+1. [horizontal divider](#horizontal-divider)
+2. [vertical divider](#vertical-divider)
+
+### Horizontal Divider
+
+Full-bleed dividers separate content into sections and span the entire length of a layout.
```

Comment:
If the section title is never changed, then the section description needs to be updated.

Reply:
Updated